src/main/scala/Configs.scala

```diff
@@ -4,6 +4,7 @@ import chisel3._
 import freechips.rocketchip.system.BaseConfig
 import freechips.rocketchip.config.{Parameters, Config}
 import freechips.rocketchip.unittest.UnitTests
+import freechips.rocketchip.diplomacy._
```

Comment:
Why do we need `diplomacy` in Configs? I don't see any other changes in this file so this is odd to me.

Reply:
Will go ahead and delete. This was left from a previous commit.